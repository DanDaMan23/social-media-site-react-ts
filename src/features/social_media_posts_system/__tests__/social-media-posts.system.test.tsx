import { render } from "@testing-library/react"
import SocialMediaPostsSystem from "../social-media-posts.system"

describe("social-media-posts.system test", () => {
  it("should render correctly", () => {
    const { baseElement } = render(<SocialMediaPostsSystem />)

    expect(baseElement).toMatchSnapshot()
  })
})
