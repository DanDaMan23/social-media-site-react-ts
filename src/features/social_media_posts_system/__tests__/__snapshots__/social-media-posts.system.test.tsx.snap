// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`social-media-posts.system test should render correctly 1`] = `
<body>
  <div>
    <h1>
      Social Media Posts System
    </h1>
    <div
      style="display: flex; justify-content: flex-end; margin: 10px;"
    >
      <button
        class="btn btn-primary"
        type="button"
      >
        Create Post
      </button>
    </div>
    <div>
      <div
        class="vstack gap-3 minBreakpoint-xs"
      />
    </div>
  </div>
</body>
`;
